### PR TITLE
style(frontend): new style for Alpha warning [GIX-2981]

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

We update the style for the Alpha warning.

### Before

Screenshot 2024-09-19 at 08 03 03

### After

Screenshot 2024-09-19 at 08 02 33

### CODE REVIEW

File: src/frontend/src/lib/i18n/en.json [hunk 1]
```diff
@@ -125,7 +125,7 @@
 	},
 	"hero": {
 		"text": {
-			"use_with_caution": "Learn more about this alpha version. Use with caution.",
+			"use_with_caution": "This is alpha version. Use with caution.",
```
Comment (peterpeterparker) [inline src/frontend/src/lib/i18n/en.json:128]: Isn't a "an" missing in the sentence? "This is an alpha" version or something?

**Review by peterpeterparker — APPROVED** [review]
Way better

File: src/frontend/src/lib/i18n/en.json (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/i18n/en.json:128]: I think so to, and in fact I wanted to have even "This is THE Alpha version..."; let me ask Max (native speaker eheh)

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/i18n/en.json]: Checked with Max: as you suggested, `This is an alpha version...` is the better version

**Pull request merged**